components/ConnectButton/index.tsx

```diff
@@ -2,7 +2,7 @@ import { ConnectButton as ConnectButtonRainbowKit } from "@rainbow-me/rainbowkit
 import { ConnectButtonProps } from "@rainbow-me/rainbowkit/dist/components/ConnectButton/ConnectButton";
 
 const ConnectButton = (props: ConnectButtonProps) => {
-  return <ConnectButtonRainbowKit {...props} />;
+  return <ConnectButtonRainbowKit chainStatus="name" {...props} />;
```

Comment:
@ECWireless how does this change the UI. I didn't notice any difference in my tests.